### PR TITLE
#18 Provide option to preserve PVCs when scaling down.

### PR DESCRIPTION
**Pull request by jhamacher-indeed**

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
I presume this is a new field in the CR?  Can you update the CRD schema to reflect the new field?

**Comment by jhamacher-indeed** [comment]
I can't figure out how to reply to your comment, @amccormick-indeed , but I updated the README.  I forgot we had a copy of the schema in two places.

**Comment by ghost** [comment]
> I can't figure out how to reply to your comment, @amccormick-indeed , but I updated the README. I forgot we had a copy of the schema in two places.

There should be a change in `rabbitmq_operator.yaml` as well -- there's an openapi v3 schema in there that's used for validation.  This won't break validation, but it'd be nice to have that "documentation" as well as type validation in there

**Comment by jhamacher-indeed** [comment]
Gotcha.  I didn't realize we had that.

File: README.md [hunk 1]
```diff
@@ -44,7 +44,8 @@ For more information on Service DNS and routing, see https://kubernetes.io/docs/
 |----------------------|-----------|-------------|
 | `rabbitMQImage`      | string    | Name of RabbitMQ image |
 | `initContainerImage` | string    | Name of initContainer image|
-| `createLoadBalancer` | boolean    | Whether to create a LoadBalancer service |
+| `createLoadBalancer` | boolean   | Whether to create a LoadBalancer service |
+| `preserveOrphanPVCs` | boolean   | When scaling down a cluster, whether to preserve "orphaned" PVCs |
```
Comment (bfink13) [inline README.md]: maybe include the default value?

**Review by bfink13 — APPROVED** [review]

**Pull request merged**